content/bn/docs/concepts/security/_index.md

```diff
@@ -0,0 +1,129 @@
+---
+title: "নিরাপত্তা"
+weight: 85
+description: >
+  ক্লাউড-নেটিভ ওয়ার্কলোডকে নিরাপত্তা রক্ষা করার প্রস্তুতির জন্য ধারণাগুলি।
+simple_list: true
+---
+
+এই কুবারনেটিসের নথিপত্রের এই অংশের উদ্দেশ্য আপনাকে ক্লাউড-নেটিভ প্রযুক্তিতে
+নিরাপত্তামুলকভাবে ওয়ার্কলোডগুলি পরিচালনা শেখানোর সাহায্য করা এবং একটি কুবারনেটিসের ক্লাস্টার
+নিরাপত্তামুলকভাবে রাখার গুরুত্বপূর্ণ দিক সম্পর্কে জানানো।
+
+কুবারনেটিসের ক্লাউড-নেটিভ বিশ্বাসের উপর ভিত্তি করে এবং ক্লাউড-নেটিভ তথ্য নিরাপত্তা সম্পর্কে ভাল অনুশীলনের 
+জন্য {{< glossary_tooltip text="CNCF" term_id="cncf" >}} থেকে 
+পরামর্শ প্রদান করে।
+
+আপনার ক্লাস্টার এবং অ্যাপ্লিকেশনগুলিকে কীভাবে সুরক্ষিত করবেন 
+সে সম্পর্কে বিস্তৃত প্রেক্ষাপটের জন্য
+[ক্লাউড নেটিভ নিরাপত্তা এবং কুবারনেটিস](/docs/concepts/security/cloud-native-security/) পড়ুন।
+
+## কুবারনেটিসের নিরাপত্তা ব্যবস্থা {#security-mechanisms}
+
+কুবারনেটিসের অনেকগুলি API এবং নিরাপত্তা নিয়ন্ত্রণ সহ অনেকগুলি নিরাপত্তা নিয়ন্ত্রণ অন্তর্ভুক্ত 
+করে, এবং তথ্য নিরাপত্তা পরিচালনার কিভাবে অংশ হতে পারে তা সম্পর্কে [নীতি](#policies) নির্ধারণের উপায় রয়েছে।
+
+### কন্ট্রোল প্লেন সুরক্ষা
+
+কোন কুবারনেটিসের ক্লাস্টারের জন্য একটি প্রধান নিরাপত্তা ব্যবস্থা হ'ল
+[কুবারনেটিসের API অ্যাক্সেসের নির্দেশিকা](/docs/concepts/security/controlling-access) প্রতিষ্ঠা করা।
+
+কুবারনেটিসের আপনার কন্ট্রোল প্লেনের মধ্যে এবং এর গ্রাহকদের মধ্যে তথ্য এনক্রিপশন প্রদানের জন্য
+TLS কনফিগার এবং ব্যবহার করার প্রত্যাশা করে।
+[কুবারনেটিসের কন্ট্রোল প্লেনে তথ্য এনক্রিপশন ইন ট্রানজিট](/docs/tasks/tls/managing-tls-in-a-cluster/)
+এবং কন্ট্রোল প্লেনের মধ্যে ডেটা জন্য [অবশিষ্ট এনক্রিপশন](/docs/tasks/administer-cluster/encrypt-data/)
+প্রয়োজনীয়; এটা আপনার নিজের ওয়ার্কলোডের ডেটা জন্য অবশিষ্ট এনক্রিপশন ব্যবহারের থেকে আলাদা করা, যা 
+আপনার জন্য ভালো একটি ধারণা হতে পারে।
```

Comment:
কুবারনেটিস আশা করে যে আপনি কন্ট্রোল প্লেনের মধ্যে এবং কন্ট্রোল প্লেন এবং এর ক্লায়েন্টদের মধ্যে 
✋ট্রানজিটে ডেটা এনক্রিপশন](/bn/docs/tasks/tls/managing-tls-in-a-cluster/) 
প্রদান করতে TLS কনফিগার করবেন এবং ব্যবহার করবেন। আপনি কুবারনেটিস কন্ট্রোল প্লেনের মধ্যে 
সংরক্ষিত ডেটার জন্য  ✋এনক্রিপশন এট রেস্ট(encryption at rest)](/bn/docs/tasks/administer-cluster/encrypt-data/) সক্ষম করতে পারেন; 
এটি আপনার নিজের ওয়ার্কলোডের ডেটার জন্য এনক্রিপশন এট রেস্ট ব্যবহার করা থেকে আলাদা, 
যা একটি ভাল আইডিয়াও হতে পারে


replace ✋ with [